### DISCUSSION TITLE
Remaining time of an audiobook in "Where you left off..." on the home screen

### DISCUSSION THREAD

**Discussion by mrburns-42** [LeoKlaus/plappa#93] [Feature Requests]
It would be great if the remaining time of an audiobook could be displayed on the home screen in the **_Where you left off..._** section. I imagine this in the red fields as an example:

![IMG_6752](https://github.com/LeoKlaus/plappa/assets/786982/6e70e209-ff48-446c-b637-e7bebe092653)

**Comment by mrburns-42** [comment]
Many thanks for the implementation. Here is a screenshot of the German version of Plappa:

![IMG_7567](https://github.com/user-attachments/assets/450c4b7a-da7f-419e-89fe-51835a62117d)

Maybe instead of HH:MM:SS just list HH:MM.

At least for me, the seconds are not decisive and without the specification, the overview could perhaps look even cleaner. In addition, an audiobook that lasts less than 10 hours is also displayed in two digits, e.g. 04 hours. Here, too, I could imagine that it would look cleaner without the 0.

**Reply by mrburns-42** [reply]
Another reason to do without the seconds would be that you could also correctly display audiobooks that are longer than 99 hours. This is very rare, but I have already seen it.

**Comment by LeoKlaus** [comment]
I've just removed the second marker, will come with the next update.

**Reply by mrburns-42** [reply]
I have just installed version 1.4.0 (40). It looks much better now. Thanks a lot for the implementation!
